app/_data/docs_nav_mesh_2.4.x.yml

```diff
@@ -164,6 +164,13 @@ inherit:
           url: /features/access-audit
         - text: MeshGlobalRateLimit (beta)
           url: /features/meshglobalratelimit
+    - title: Guides
+      action: insert
+      index: -3
+      icon: /assets/images/icons/documentation/icn-enterprise-blue.svg
+      items:
+        - text: Use OPA to validate JWT tokens
+          url: /guides/jwt-validation/
```

Comment:
@fabianrbz Do you know why this isn't appearing correctly in the preview nav? It wasn't showing up for me locally. I copied what we used for the Enterprise section, but I must've done something wrong.